src/Microsoft.AspNet.Mvc.ModelBinding/ModelMetadataTypeAttribute.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace Microsoft.AspNet.Mvc
```

Comment:
We are open on getting a better name for the attribute but we decided to go with a new attribute.